app/views/ExposureHistory/ExposureDatumDetail.tsx

```diff
@@ -47,11 +48,14 @@ interface PossibleExposureDetailProps {
 const PossibleExposureDetail = ({
   datum: { date, possibleExposureTimeInMin, currentDailyReports },
 }: PossibleExposureDetailProps) => {
+  const exposureDurationText = TimeHelpers.durationMsToString(
+    possibleExposureTimeInMin * 60 * 1000,
+  );
   const navigation = useNavigation();
   const exposureDate = dayjs(date).format('dddd, MMM DD');
-  const exposureTime = `Possible Exposure Time: ${possibleExposureTimeInMin}`;
+  const exposureTime = `Possible Exposure Time: ${exposureDurationText}`;
   const dailyReports = `Current daily reports: ${currentDailyReports}`;
-  const explainationContent = `For ${possibleExposureTimeInMin} consecutive minutes, your phone was within 10 feet of someone who later received a confirmed positive COVID-19 diagnosis.`;
+  const explainationContent = `For ${exposureDurationText}, your phone was within 10 feet of someone who later received a confirmed positive COVID-19 diagnosis.`;
```

Comment:
small type fwiw, ~explaination~ explanation

Reply:
also, I may be reading this incorrectly, but when the string is interpolated, won't this read: "for 40, your phone was within..." - i.e. missing "minutes"?

Reply:
the dayjs humanize method which is used for this provides minutes, hours, days, etc.